styling.css

```diff
@@ -631,3 +631,123 @@ li::marker {
 }
 
 /* end of about section */
+
+/* start to contact form */
+
+.contact_form {
+  background-color: #fffffffd;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 399px;
+  justify-self: center;
+}
+
+.foot_section {
+  background-color: white;
+  width: 399px;
+  justify-self: center;
+}
+
+.contact_mssg {
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
```

Comment:
[OPTIONAL]
- [ ] Please center your `form` to follow Figma's design (see image)
- [ ] Kindly change the color of the lines to follow Figma's design (see image)
This is part of the requirement however since the previous reviewer didn't mention it, I put them as optional. But I highly recommend you put them into consideration.
### Your design 
![Screenshot from 2023-04-05 08-22-35](https://user-images.githubusercontent.com/91142432/230032408-9f630a9e-55fa-48c7-bbae-e90bdfc51e7a.png)

### Figma design
![Screenshot from 2023-04-05 08-56-04](https://user-images.githubusercontent.com/91142432/230032623-bf5b6b9b-c276-4b62-ade0-66fc5c8bc9d0.png)